{{cookiecutter.repository_name}}/.github/workflows/pre-commit.yml

```diff
@@ -2,24 +2,16 @@
 {% raw %}
 on:
   pull_request:
-  merge_group:
 
 jobs:
   pre-commit:
     name: "pre-commit"
     runs-on: ubuntu-latest
     steps:
-      - name: "Do not ignore pre-commit failures on PRs"
-        if: github.event_name != 'merge_group'
-        run: echo "IGNORE_FAILURE=false" >> $GITHUB_ENV
-        shell: bash
-      - name: "Ignore pre-commit failures on gh merge queues"
-        if: github.event_name == 'merge_group'
-        run: echo "IGNORE_FAILURE=true" >> $GITHUB_ENV
-        shell: bash
-
-      - uses: SonarSource/gh-action_pre-commit@dfd365240ac6bc71b9912ff8f231a7543214f1b2 # 0.0.2
+      - uses: SonarSource/gh-action_pre-commit@f04ea4aa921469a3f203f82f8965d3a308f59d91 # 0.0.7
```

Comment:
What do you think about using `SonarSource/gh-action_pre-commit@<main-version>` instead of pinning the version? Most of the teams still need to start using dependency automation.

Reply:
I think this is something we have to automate in this same repository, we should not provide them non working example IMHO. 

This fix will simply reduce a few the PRs temporarily as at the moment as soon as we create a new Repo, then renovate a new PR to update this ... (noise)

Out of scope for this PR